mitodl/micromasters
# Renamed inner to dashboard, and outer to public.

#### What are the relevant tickets?

Related to #52 and #75.

#### What's this PR do?

Renames templates, javascript to reflect our nomenclature for the parts of the application.

#### Where should the reviewer start?

See [Files changed](https://github.com/mitodl/micromasters/pull/76/files)

#### How should this be manually tested?

Visit `/`, then visit `/dashboard`; nothing should function differently from master.

#### Any background context you want to provide?

We should keep files and routes and parts of the application named appropriately.

#### Screenshots (if appropriate)

N/A

#### What GIF best describes this PR or how it makes you feel?

![butterfly](http://teacher.scholastic.com/activities/explorations/bug/images/butterfly.jpg)
